examples/learn/calls/multi_modal_outputs/openai/base_message_param.py

```diff
@@ -0,0 +1,36 @@
+import io
+import wave
+
+from pydub.playback import play
+
+from mirascope.core import openai, BaseMessageParam
+from mirascope.core.openai import AudioSegment
+
+
+@openai.call(
+    "gpt-4o-audio-preview",
+    call_params={
+        "audio": {"voice": "alloy", "format": "wav"},
+        "modalities": ["text", "audio"],
+    },
+)
+def recommend_book(genre: str) -> list[BaseMessageParam]:
+    return [BaseMessageParam(role="user", content=f"Recommend a {genre} book")]
+
+
+response = recommend_book(genre="fantasy")
+
+print(response.audio_transcript)
+
+if audio := response.audio:
+    audio_io = io.BytesIO(audio)
+
+    with wave.open(audio_io, "rb") as f:
+        audio_segment = AudioSegment.from_raw(
```

Comment:
can we not just do `AudioSegment.from_raw(audio_io)`?

Reply:
We can't play without parameters like `sample_width`

```python
    audio_io = io.BytesIO(audio)

    with wave.open(audio_io, "rb") as f:
        audio_segment = AudioSegment.from_raw(audio_io)
```

```bash
Traceback (most recent call last):
  File "/Users/koudai/PycharmProjects/mirascope/examples/learn/calls/multi_modal_outputs/openai/messages.py", line 29, in <module>
    audio_segment = AudioSegment.from_raw(audio_io)
  File "/Users/koudai/PycharmProjects/mirascope/.venv/lib/python3.10/site-packages/pydub/audio_segment.py", line 812, in from_raw
    return cls.from_file(file, 'raw', sample_width=kwargs['sample_width'], frame_rate=kwargs['frame_rate'],
KeyError: 'sample_width'
```